.templates/mopidy/mopidy.conf

```diff
@@ -0,0 +1,28 @@
+[core]
+data_dir = /var/lib/mopidy
+
+[local]
+media_dir = /var/lib/mopidy/media
+
+[audio]
+output = tee name=t ! queue ! autoaudiosink t. ! queue ! udpsink host=0.0.0.0 port=5555
+
+[m3u]
+playlists_dir = /var/lib/mopidy/playlists
+
+[http]
+hostname = 0.0.0.0
+
+[mpd]
+hostname = 0.0.0.0
+
+[spotify]
+username=USERNAME
+password=PASSWORD
+
+[gmusic]
```

Comment:
Google music is no longer a thing and is being sunsetted this month.

Reply:
Yes, I am still working on getting mopidy container up & running so please ignore mopidy related changes in repo.

Reply:
Updated mopidy configuration, removed GMusic from configuration and switched to Dockerfile build image. Working on my RPi 4. (ToDo: Push well-tested image to docker hub and change service configuration.)